manager/src/main/java/com/redhat/service/smartevents/processingerrors/ProcessingErrorHandler.java

```diff
@@ -78,4 +83,10 @@ private Map<String, String> parseHeaders(Headers headers) {
     private JsonNode parsePayload(String payload) throws JsonProcessingException {
         return payload == null ? null : objectMapper.readTree(payload);
     }
+
+    @Scheduled(cron = "{event-bridge.processing-errors.cleanup.schedule}")
+    void cleanup() {
```

Comment:
```suggestion
    private void cleanup() {
```

Reply:
Honestly here I followed the [quarkus guide](https://quarkus.io/guides/scheduler#standard-scheduling) where all the methods annotated with `@Scheduled` have no visibility modifier. I'd like to stick with this approach.